apps/judicial-system/web/src/routes/Court/Indictments/Advocates/Advocates.tsx

```diff
@@ -50,23 +52,36 @@ const HearingArrangements = () => {
     >
       <PageHeader title={formatMessage(titles.court.indictments.defender)} />
       <FormContentContainer>
-        <PageTitle>{formatMessage(m.title)}</PageTitle>
+        <PageTitle>{formatMessage(strings.title)}</PageTitle>
         <CourtCaseInfo workingCase={workingCase} />
         <Box component="section" marginBottom={5}>
           <AlertMessage
-            message={formatMessage(m.alertBannerText)}
+            message={formatMessage(strings.alertBannerText)}
             type="info"
           />
         </Box>
-        <Box component="section" marginBottom={10}>
+        <Box component="section" marginBottom={hasCivilClaimants ? 5 : 10}>
           <SectionHeading
-            title={formatMessage(m.selectDefenderHeading)}
+            title={formatMessage(strings.selectDefenderHeading)}
             required
           />
           {workingCase.defendants?.map((defendant, index) => (
             <SelectDefender defendant={defendant} key={index} />
           ))}
         </Box>
+        {hasCivilClaimants && (
+          <Box component="section" marginBottom={10}>
+            <SectionHeading title={formatMessage(strings.civilClaimants)} />
+            {workingCase.civilClaimants?.map((civilClaimant, index) => (
+              <Box component="section" marginBottom={5}>
+                <SelectCivilClaimantAdvocate
+                  civilClaimant={civilClaimant}
+                  key={index}
```

Comment:
The `key` prop is not used in `SelectCivilClaimantAdvocate`. Also, the parent `Box` does not have a key prop set, which will result in a `Unique key` warning. Please move the key prop to the `Box` above and consider setting it to something more unique, like `civilClaimant.id`.